tools/tune/dmic/dmic_batch.m

```diff
@@ -27,25 +27,31 @@
 prm.pdm(2).polarity_mic_b = 0;
 prm.pdm(2).clk_edge = 0;
 prm.pdm(2).skew = 0;
+prm.vfn = {};
 
 %% Design
 prm.pdmclk_min = 2.4e6;
 prm.pdmclk_max = 4.8e6;
 
+%% alsa-utils
+if exist('include_alsa_utils') == 7
```

Comment:
I believe there is a second argument missing!

Reply:
yes, using 'dir' should be faster so good to do.